### PR TITLE
Add WIP guide for Client Communication

### PR DESCRIPTION
**Pull request by laurenfackler**
Rough draft!

### CODE REVIEW

**Comment by ninzaburoz** [comment]
I really like what you have started! If anything, I would add that if an email is going to be long or consist of necessary documentation, that we try to bullet point things for the client. 

With Soloshot, it's helped with ticket building and with tracking work if we provided the bullet points/ list of issues / list of questions for them to copy, answer and send back (or make tickets). Short emails are great - I'm a fan of them myself - but sometimes they aren't possible and a call can't be scheduled within a reasonable amount of time~ So, explain and leave bullet points! :D (just an idea!!)

**Comment by Jacob-Ernst** [comment]
Dig all of this guys! 

One thing that I think can cause a lot of problems is unexpected reactions between team members during meetings. For example, whenever one team member is shocked/surprised by something another team member says. There will definitely be differences of opinion between Grokers, but it's much better to try to communicate about these things before client meetings/demos. It looks really bad when the team doesn't seem on the same page. I think everyone has seen this to some degree, and that's why I think pre-meeting communication amongst the team is invaluable.

**Comment by laurenfackler** [comment]
Awesome feedback @Jacob-Ernst and @ninzaburoz , thank you! Love it. I'll incorporate both your suggestions into the next push :)

File: client-communication/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Shorter emails are generally eaiser to digest. Some clients may miss important points if an email is long.
+- Consider scheduling a call instead, if an email is lengthy.
+- If you can't avoid sending a long email, do your best to break it up; bullet points/lists work well.
+- Including screen shots in your emails can be really helpful; OS X Preview has some nice annotate tools.
+
+![Annotating in OS X Preview](annotate.gif "Annotating in OS X Preview")
+
+## Phone Calls
+
+- Be cautious with giving out a personal phone number to a client.
```
Comment (JDStraughan) [inline client-communication/README.md]: Add guide to using TalkRoute for calling using office phone number.

http://talkroute.com/

**Pull request merged**